### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by awav**

### CODE REVIEW

File: gpflow_logo.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...14.64,20.8Z"/><polygon class="cls-1" points="389.89 25.57 373.44 81.09 370.22 81.09 355.07 29.92 339.91 81.09 336.69 81.09 320.24 25.57 311.13 25.57 329.76 88.83 344.05 88.83 355.07 52.64 366.08 88.83 380.37 88.83 399 25.57 389.89 25.57"/><polygon class="cls-1" points="371.83 77.81 387.31 25.57 378.2 25.57 371.5 54.01 363.31 25.57 363.3 25.57 356.36 25.57 371.83 77.81"/><polygon class="cls-1" points="338.3 77.81 353.77 25.57 346.83 25.57 346.82 25.57 338.63 54.01 331.93 25.57 322.82 25.57 338.3 77.81"/><path class="cls-2" d="M145.9,41.94a4.32,4.32,0,1,0,.9,3.19A4.34,4.34,0,0,0,145.9,41.94Z"/><path class="cls-2" d="M26.79,19.13a4.32,4.32,0,1,0,.9,3.19A4.33,4.33,0,0,0,26.79,19.13Z"/><path class="cls-2" d="M112.59,11.93a4.33,4.33,0,1,0-3.4,7,4.33,4.33,0,0,0,4.29-3.81A4.33,4.33,0,0,0,112.59,11.93Z"/><path class="cls-2" d="M71,67.72a4.32,4.32,0,1,0,.9,3.19A4.34,4.34,0,0,0,71,67.72Z"/><path class="cls-2" d="M192.77,11.25a4.33,4.33,0,1,0-3.4,7,4.33,4.33,0,0,0,4.29-3.81A4.33,4.33,0,0,0,192.77,11.25Z"/><path class="cls-2" d="M295.34,71.16a4.32,4.32,0,1,0,.9,3.19A4.35,4.35,0,0,0,295.34,71.16Z"/><path class="cls-2" d="M265.11,37.91a4.31,4.31,0,1,0,.9,3.2A4.35,4.35,0,0,0,265.11,37.91Z"/></svg>
```
Comment (condnsdmatters) [inline gpflow_logo.svg]: could you move these image files to the `doc/source/_static`

Comment (awav) [reply gpflow_logo.svg]: done

File: README.md [hunk 2]
```diff
@@ -18,33 +18,41 @@ The full list of [contributors](http://github.com/GPflow/GPflow/graphs/contribut
...
+
+## Getting Started with GPflow 2.0
 
-in a check-out of the `develop` branch of the GPflow github repository.
+There is an ["Intro to GPflow 2.0"](https://github.com/GPflow/GPflow/blob/develop/doc/source/notebooks/intro_to_gpflow2.ipynb) Jupyter notebook. Check it out for details.
 
+- **GPflow 1.0**
 
-## Getting Started
+  *We stop the support for GPflow based on TensorFlow 1.0. We accept bug fixes only to GPflow   1.0 in **develop-1.0** branch. Latest available release is [v1.5.1](https://github.com/GPflow/GPflow/releases/tag/v1.5.1).*
```
Comment (condnsdmatters) [inline README.md]:
```suggestion
  *We have stopped development and support for GPflow based on TensorFlow 1.0. We now accept only bug fixes to GPflow 1.0 in the **develop-1.0** branch. The latest available release is [v1.5.1](https://github.com/GPflow/GPflow/releases/tag/v1.5.1).*
```

Comment (condnsdmatters) [reply README.md]: tempted to just commit this, but you should do it. happy to accept with this done.

**Pull request merged**